setup.py

```diff
@@ -50,15 +50,10 @@ def get_packages(package):
     long_description_content_type="text/markdown",
     author="Tom Christie",
     author_email="tom@tomchristie.com",
-    package_data={"httpcore": ["py.typed"]},
     packages=get_packages("httpcore"),
     include_package_data=True,
     zip_safe=False,
-    install_requires=[
-        "h11>=0.8,<0.10",
-        "h2==3.*",
-        "sniffio==1.*",
-    ],
+    install_requires=["h11>=0.8,<0.10", "h2==3.*", "sniffio==1.*",],
```

Comment:
Black tried to sneak this one in ;-)

```suggestion
    install_requires=["h11>=0.8,<0.10", "h2==3.*", "sniffio==1.*"],
```